### PR TITLE
Add scope parameter to KeycloakInitOptions

### LINKED ISSUE #11824: keycloak-js: Expose scope in KeycloakInitOptions
### Description

keycloak-js `init()` supports scope parameter as implemented in [keycloak.js:151](https://github.com/keycloak/keycloak/blob/3bcdb44d6a2d365c4a42c6d91dd7cb555897db49/adapters/oidc/js/src/keycloak.js#L151).

However, the corresponding parameter is missing from the interface definition [keycloak.d.ts:55](https://github.com/keycloak/keycloak/blob/main/adapters/oidc/js/dist/keycloak.d.ts#L55).

The parameter should be added to the interface.

### Discussion

_No response_

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by epuronta**
keycloak-js init() supports scope parameter as implemented in [keycloak.js:151](https://github.com/keycloak/keycloak/blob/3bcdb44d6a2d365c4a42c6d91dd7cb555897db49/adapters/oidc/js/src/keycloak.js#L151).

However, the corresponding parameter is missing from the interface definition [keycloak.d.ts:55](https://github.com/keycloak/keycloak/blob/main/adapters/oidc/js/dist/keycloak.d.ts#L55).

The parameter should be added to the interface.

Closes #11824

### CODE REVIEW

**Review by abstractj — APPROVED** [review]
LGTM @jonkoops waiting for your approval

**Review by jonkoops — APPROVED** [review]

**Pull request merged**
